packages/babel-preset-env/src/normalize-options.js

```diff
@@ -35,7 +35,7 @@ const expandIncludesAndExcludes = (
   plugins: Array<string | RegExp> = [],
   type: string,
 ): Array<string> => {
-  if (plugins.length === 0) return plugins;
+  if (plugins.length === 0) return [];
```

Comment:
Couldn't return `plugins` here because the output is different type than the input. I assumed that we're safe to return just an empty array here, because technically we don't accept any other types.